src/locales/en.json

```diff
@@ -103,7 +103,11 @@
 			"login_placeholder": "CAS-UTC or email",
 			"password_label": "Password :",
 			"password_placeholder": "Connection password",
-			"bad_login_password": "Wrong identifiants",
+			"bad_login_password": "Wrong credentials",
+			"i_am_ext": "I am extern",
+			"i_am_cas": "I am CAS-UTC",
+			"wrong_ext": "Check that you gave the right email adress you gave for your external contribution.",
+			"wrong_cas": "Check that you correctly wrote down your CAS-UTC login and password.",
```

Comment:
Only one simple message would be fine: "Check that you correctly wrote your credentials."